README.md

```diff
@@ -40,61 +40,62 @@ distributions with the [releases][] (but please host your own copy).
-  var type = avro.parse({
-    name: 'Pet',
-    type: 'record',
-    fields: [
-      {name: 'kind', type: {name: 'Kind', type: 'enum', symbols: ['CAT', 'DOG']}},
-      {name: 'name', type: 'string'}
-    ]
-  });
-  var buf = type.toBuffer({kind: 'CAT', name: 'Albert'}); // Encoded buffer.
-  var val = type.fromBuffer(buf); // {kind: 'CAT', name: 'Albert'}
-  ```
+```javascript
+const type = avro.parse({
+  name: 'Pet',
+  type: 'record',
+  fields: [
+    {name: 'kind', type: {name: 'Kind', type: 'enum', symbols: ['CAT', 'DOG']}},
+    {name: 'name', type: 'string'}
+  ]
+});
+const buf = type.toBuffer({kind: 'CAT', name: 'Albert'}); // Encoded buffer.
+const val = type.fromBuffer(buf); // {kind: 'CAT', name: 'Albert'}
+```
 
 + Infer a value's type and encode similar values:
 
-  ```javascript
-  var val = {city: 'Cambridge', zipCodes: ['02138', '02139'], visits: 2};
-  var type = avro.infer(val);
-  // We can now encode the value:
-  var buf = type.toBuffer(val);
-  // And also any values with a matching structure:
-  var bufs = [
-    type.toBuffer({city: 'Seattle', zipCodes: ['98101'], visits: 3}),
-    type.toBuffer({city: 'NYC', zipCodes: [], visits: 0})
-  ];
-  ```
+```javascript
+const val = {city: 'Cambridge', zipCodes: ['02138', '02139'], visits: 2};
+const type = avro.infer(val);
+// We can now encode the value:
+const buf = type.toBuffer(val);
+// And also any values with a matching structure:
+const bufs = [
+  type.toBuffer({city: 'Seattle', zipCodes: ['98101'], visits: 3}),
+  type.toBuffer({city: 'NYC', zipCodes: [], visits: 0})
+];
+```
 
 + Get a [readable stream][readable-stream] of decoded values from an Avro
   container file:
 
-  ```javascript
-  avro.createFileDecoder('./values.avro')
-    .on('metadata', function (type) { /* `type` is the writer's type. */ })
-    .on('data', function (val) { /* Do something with the decoded value. */ });
-  ```
+```javascript
+avro.createFileDecoder('./values.avro')
+  .on('metadata', type => { /* `type` is the writer's type. */ })
+  .on('data', val => { /* Do something with the decoded value. */ });
+```
 
 + Implement a TCP server for an [IDL-defined][idl] protocol:
 
-  ```javascript
-  avro.assemble('./Ping.avdl', function (err, attrs) {
-    // Generate the protocol and attach a handler for `ping` messages:
-    var protocol = avro.parse(attrs)
-      .on('ping', function (req, ee, cb) { cb(null, 'pong'); });
-    // Respond on any incoming connection:
-    require('net').createServer()
-      .on('connection', function (con) { protocol.createListener(con); })
-      .listen(8000);
-  });
-  ```
-
+```javascript
+avro.assemble('./Ping.avdl', (err, attrs) => {
+  // Generate the protocol and attach a handler for `ping` messages:
+  const protocol = avro.parse(attrs)
+    .on('ping', (req, ee, cb) => {
+      cb(null, 'pong');
+    });
+  // Respond on any incoming connection:
+  require('net').createServer()
+    .on('connection', con => { protocol.createListener(con); })
```

Comment:
Since you've split the callback above over multiple lines, can you split this one as well? (And wrap `con` inside parens.)